### ISSUE TITLE
Automobilista crash to desktop with D9VK

### ISSUE THREAD

**Issue by leillo1975** [Joshua-Ashton/d9vk#433]
With D9VK, The game goes to desktop after a some time with a lot of Stutter. You can see a video here:
https://youtu.be/k_122qOl2UY
Without D9VK the game don't crash to desktop .

Please describe your issue as accurately as possible. If you run into a problem with a binary release, make sure to test with latest `master` as well.

### Software information
Autombilista (Steam ID 431600)

### System information
- GPU: Nvidia GeForce GTX-1060-6GB
- Driver: 430
- Wine version: Proton 4.11-7
- D9VK version: 0.22

**Comment by Joshua-Ashton** [comment]
Please test with `master` builds are here: https://git.froggi.es/joshua/d9vk/-/jobs

You also ignored and deleted the most important part of the issue template. If I didn't want that information I wouldn't put it on there.

**Comment by leillo1975** [comment]
Seems that with the last master (4ad9c589), the game works great and don't crash to desktop. I only could test about 2 minutes now, but this night I will play more time to confirm it.

**Comment by leillo1975** [comment]
After playing more time I can confirm that with the version 4ad9c58, the game don't crash. I tried the new 0.30 "official" version and with it the game continues crashing.

**Comment by Joshua-Ashton** [comment]
Well nothing after that can cause a crash ever.

**Comment by Joshua-Ashton** [comment]
Unless it was the rebase, what happens with the build for the ZERO fix on git.froggi.es jobs

**Comment by jrugia** [comment]
Could this be another case of the game running out of virtual address space, in which case the first thing I suggest is to patch the game to be Large Address Aware.

**Comment by leillo1975** [comment]
Seems that now it don't crash with Proton 4.11-9. This happens now that I've managed to make a trace.... I upload it to Mega:
https://mega.nz/#!ABwQjA4K!2osCv_WhXaALoTJvDGdx-qzClx9e783RCtH0C73DGKY

I see that there are stuttering and FPS are low (without D9VK I think is more higher)

**Comment by leillo1975** [comment]
It crash to desktop again. I found this problem when you crash your car in the pits. I found another problem. The colors, especially in menus, is very contrasted and strong.

**Comment by Joshua-Ashton** [comment]
Hey, I can't repro this on the trace:

https://store.steampowered.com/app/431600/Automobilista/ is the game right?

**Comment by leillo1975** [comment]
Yes, is this, but some days ago I was playing a lot of time with the last version without problems. Take a look to the video, it works great:
https://youtu.be/Xs3Pzn5ee5k
